pages/login.tsx

```diff
@@ -19,6 +19,11 @@ function Login({ ssoPassEmploiEstActif, isFromEmail }: LoginProps) {
   const [errorMsg, setErrorMsg] = useState('')
   const router = useRouter()
 
+  async function handleSubmit(event: any, provider?: string) {
+    event.preventDefault()
+    signin(provider)
```

Comment:
```suggestion
   await signin(provider)
```
nitpick: (relevé par mon IDE)